### PR TITLE
Fix sibling service link on fence date

### PR DESCRIPTION
**Pull request by felipeparaujo**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: docs/day-for-tv-and-radio.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+What's a day in the schedules page?
+===================================
+
+One of the many peculiarities of /programmes is that what a day constitutes varies depending on the outlet.
+
+A day for Radio outlets starts at 12:00:00 and ends at 23:59:59. Normal stuff.
```
Comment (votemike) [inline docs/day-for-tv-and-radio.md]: Not true. It runs for 30 hours from 00:00:00 to 05:59:59 the next day.

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: docs/day-for-tv-and-radio.md (hunk 1 above)
Comment (BPScott) [inline docs/day-for-tv-and-radio.md]: Yepyep, see the comments at the top of BroadcastDay

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**